src/components/help/help.spec.tsx

```diff
@@ -1,17 +1,22 @@
 import React from "react";
 import { act } from "react-dom/test-utils";
-import { shallow, mount } from "enzyme";
+import { shallow, mount, ShallowWrapper, ReactWrapper } from "enzyme";
 import Icon from "components/icon";
-import Help from "./help.component";
+import Help, { HelpProps } from "./help.component";
 import { rootTagTest } from "../../__internal__/utils/helpers/tags/tags-specs";
 import StyledHelp from "./help.style";
 import Tooltip from "../tooltip";
 import { testStyledSystemMargin } from "../../__spec_helper__/test-utils";
 
 jest.mock("@tippyjs/react/headless");
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
What do you think about using `mount` only here instead of allowing both it and `shallow`? This would eliminate the need to suppress the `any` warning

Reply:
My personal preference is to leave it as is for now as there's about half the cases that are using shallow which offers a degree of optimisation when running the test file and we'll also be migrating to RTL anyway so the less changes the better IMHO. However, I'm happy to make the change if the second reviewer also agrees with you 👍

Reply:
alternatively you could just return the JSX element and use `shallow` and `mount` directly in tests. In my opinion we shouldn't set the renderer in that method in the first place:
`shallow(renderHelp({ type: "home", tooltipFlipOverrides: ["top", "bottom"] }));